VitalHealthConnect/src/main/java/io/tryvital/vitalhealthconnect/VitalHealthConnectManager.kt

```diff
@@ -222,6 +205,10 @@ class VitalHealthConnectManager private constructor(
         val userId = checkUserId()
 
         try {
+            // TODO: VIT-2924 Move userId management to VitalClient
+            // TODO: VIT-2944 VitalClient to keep track of created connected sources in SharedPreferences.
+            vitalClient.createConnectedSource(ManualProviderSlug.HealthConnect, userId = userId)
```

Comment:
This is now part of the sync process (= the iOS SDK "automatic" mode). The manual "Link Provider" button is removed from the example app.